### ISSUE TITLE
[Bug]: WellDataModule normalization not applied to val and test data

### ISSUE THREAD

**Issue by radka-j** [PolymathicAI/the_well#68]
### Describe the issue:

Hi!

I noticed that in the `WellDataModule` the `use_normalization` and `normalization_type` parameters are only passed to `WellDataset` when creating the train data but not when creating the validation or test datasets. This seems to have been removed in [this commit](https://github.com/PolymathicAI/the_well/commit/4866ff299ac177665d0fe55e05db98fd254e0012#diff-26832a0cba027ecbd6262cfe63865ebf09ed2c6e57521458314da28abf4af6bc).

Is this a bug or is this handled somewhere else that I might have missed?

### Code to reproduce the issue:

```python
See implementation of the `WellDataModule` [here](https://github.com/PolymathicAI/the_well/blob/master/the_well/data/datamodule.py). Specifically the `__init__` method where `self.train_dataset`, `self.val_dataset`, `self.rollout_val_dataset` and `self.test_dataset` are created.
```

### Version

Latest available on GitHub

### Environment

n/a

### Context for the issue:

n/a

**Comment by radka-j** [comment]
Thank you for pointing me to the right part of the codebase! I was trying to figure out if the normalisation happens elsewhere but couldn't find it easily so thought I'd just check here.

**Comment by maxxxzdn** [comment]
Hi folks, thanks for addressing the issue!

I just trained an FNO on `active_matter` with the default hyperparameters, and it scored 0.15 VRMSE pretty comfortably after just 1h of training. Therefore, I wonder, am I doing something wrong, or are the values reported in Table 2 wrong?

**Comment by mikemccabe210** [comment]
Hi @maxxxzdn - the repo has had a number of changes and bug fixes since it was originally released so the table is outdated and what you're reporting is consistent with what we're seeing in current runs. We have a team member gradually re-running things on the current version of the repo to account for the changes/fixes and we're planning to update the issue (#49), HF, and the table on arxiv once he's finished to get everything in final sync for new users.

In general, now that the data has been out long enough to see real use, even once those are fixed, I'd mostly only recommend the baselines for sanity checking. The original baselines were run in an intentionally naive time-boxed setting due to having to budget for HP tuning across so many datasets, and most of the work we've seen using the data since then has usually employed much higher tuning/training budgets for the Well datasets they've used.
